Microservices.PlatformService/Profiles/PlatformProfiles.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using Microservices.PlatformService.Dtos;
+using Microservices.PlatformService.Models;
+
+namespace Microservices.PlatformService.Profiles
+{
+    public class PlatformProfiles : Profile
+    {
+        public PlatformProfiles()
+        {
+            CreateMap<PlatformReadDto, Platform>()
```

Comment:
What it the reason of introducing PlatformReadDto if it contains exactly the same properties as domain model Platform?

Reply:
It's used validation attributes.